test/common/3scale_crudl.go

```diff
@@ -5,6 +5,7 @@ import (
 	"crypto/rand"
 	"encoding/json"
 	"fmt"
+	"golang.org/x/net/context"
```

Comment:
why adding "golang.org/x/net/context" instead of just "context"

Reply:
done